### ISSUE TITLE
Popover bug in bootstrap.js - 3.3.5

### ISSUE THREAD

**Issue by kameas** [twbs/bootstrap#17550]
1. click on button popover
2. click on body
3. click on button popover (3.3.4 - 1 click, 3.3.5 - 2 click)

https://jsfiddle.net/2jne44zh/1/ - its work on 3.3.4
https://jsfiddle.net/m6xdqg8y/ - now its bug on 3.3.5

**Comment by twbs-lmvtfy** [comment]
Hi @kameas!

You appear to have posted a live example (https://fiddle.jshell.net/2jne44zh/1/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 16, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.5 ; saw what appears to be usage of Bootstrap 3.0.2
* line 20, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.5 ; saw what appears to be usage of Bootstrap 3.3.4

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by twbs-lmvtfy** [comment]
Hi @kameas!

You appear to have posted a live example (https://fiddle.jshell.net/m6xdqg8y/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 16, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.5 ; saw what appears to be usage of Bootstrap 3.0.2

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
And you're mixing different versions of Bootstrap's CSS and JavaScript, which is also invalid; their versions must always match.
If there's still an issue after resolving that, please post revised JS Fiddles and I'll be happy to reopen this issue.

**Comment by kameas** [comment]
update:
https://jsfiddle.net/2jne44zh/2/ - 3.3.4
https://jsfiddle.net/m6xdqg8y/1/ - 3.3.5

**Comment by twbs-lmvtfy** [comment]
Hi @kameas!

You appear to have posted a live example (https://fiddle.jshell.net/2jne44zh/2/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 16, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.5 ; saw what appears to be usage of Bootstrap 3.3.4
* line 20, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.5 ; saw what appears to be usage of Bootstrap 3.3.4

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Duplicate of #16732.
